org.libsdl.Maelstrom.yml

```diff
@@ -0,0 +1,43 @@
+app-id: org.libsdl.Maelstrom
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: maelstrom
+rename-icon: maelstrom
+rename-desktop-file: maelstrom.desktop
+rename-appdata-file: maelstrom.appdata.xml
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
sdl support wayland ? I though it was still experimental / buggy?

Reply:
On other SDL2 games we enabled it.